homeassistant/components/climate/__init__.py

```diff
@@ -243,6 +261,29 @@ def async_away_mode_set_service(service):
         schema=SET_AWAY_MODE_SCHEMA)
 
     @asyncio.coroutine
+    def async_hold_mode_set_service(service):
+        """Set hold mode on target climate devices."""
+        target_climate = component.async_extract_from_service(service)
+
+        hold_mode = service.data.get(ATTR_HOLD_MODE)
+
+        if hold_mode is None:
```

Comment:
I find that this same pattern is used in async_away_mode_service, async_aux_heat_set_service, async_humidity_set_service, async_fan_mode_set_service, and async_operation_set_service. Should these checks be eliminated there as well for consistency?